### PR TITLE
Implements Equivalent Object property Axiom with test cases.

### PR DESCRIPTION
**Pull request by skapoe**
- Implements equivalent object property axiom with test cases.

Signed-off-by: skapoe <sourabhkapoor1512@gmail.com>

### CODE REVIEW

File: owl2nl/src/main/java/org/aksw/owl2nl/OWLAxiomConverter.java [hunk 1]
```diff
@@ -191,8 +189,33 @@ public void visit(OWLSubObjectPropertyOfAxiom axiom) {
...
+
+
+				NLGElement Equi2ObjPropertyElement = peConverter.asNLGElement(ObjProp.get(j), false);
+				logger.debug("Equivalent Property 2: " + realiser.realise(Equi2ObjPropertyElement));
+				SPhraseSpec clause = nlgFactory.createClause(EquiObjPropertyElement, "be equivalent to", Equi2ObjPropertyElement);
+				nl = realiser.realise(clause).toString();
+				logger.debug(axiom + " = " + nl);
+
+				/*	OWLSubObjectPropertyOfAxiom subObjPAxiom = df.getOWLSubObjectPropertyOfAxiom(
+						ObjProp.get(i),
```
Comment (AmbujMondal) [inline owl2nl/src/main/java/org/aksw/owl2nl/OWLAxiomConverter.java]: remove unwanted commented code

File: owl2nl/src/main/java/org/aksw/owl2nl/OWLAxiomConverter.java [hunk 2]
```diff
@@ -191,8 +189,33 @@ public void visit(OWLSubObjectPropertyOfAxiom axiom) {
 
 	@Override
 	public void visit(OWLEquivalentObjectPropertiesAxiom axiom) {
+		Set<OWLObjectPropertyExpression> ObjProperty = axiom.getProperties();
+		List<OWLObjectPropertyExpression> ObjProp = new LinkedList<>(ObjProperty);
```
Comment (AmbujMondal) [inline owl2nl/src/main/java/org/aksw/owl2nl/OWLAxiomConverter.java]: ArrayList is better for storing and accessing data while doubly LinkedList is better for manipulating data.
`https://www.javatpoint.com/difference-between-arraylist-and-linkedlist`

**Review by AmbujMondal — COMMENTED** [review]
Good Work. 👍🏼

File: owl2nl/src/test/java/org/aksw/owl2nl/OWLAxiomConverterTest.java [hunk 3]
```diff
@@ -138,6 +142,14 @@ public void testFunctionalObjectProperty() throws OWLAxiomConversionException {
         Assert.assertEquals("⊤ ⊑ ≤ 1 hasMother", axiom.toString());
         Assert.assertEquals("everything is something that has at most 1 mother", text);
     }
+    @Test
```
Comment (AparnaSivaprasad) [inline owl2nl/src/test/java/org/aksw/owl2nl/OWLAxiomConverterTest.java:146]: Please add a blank line between consecutive methods.

**Pull request closed without merging**